NitroxClient/MonoBehaviours/Gui/MainMenu/MainMenuMultiplayerPanel.cs

```diff
@@ -22,7 +23,8 @@ public class MainMenuMultiplayerPanel : MonoBehaviour
         private GameObject multiplayerButton;
         private Transform savedGameAreaContent;
         public GameObject SavedGamesRef;
-        public string SERVER_LIST_PATH = Path.Combine(".", "servers");
+        private static readonly Uri url = new Uri(System.Reflection.Assembly.GetAssembly(typeof(uGUI)).CodeBase);
```

Comment:
Use the `Location` property instead: https://docs.microsoft.com/en-us/dotnet/api/system.reflection.assembly.codebase?view=netcore-3.1